### DISCUSSION TITLE
Sub-challenges feature?

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#383] [Q&A] [answered]
We currently have some sub-challenges arising within our innovation-communities. For example in the " digital wallets" challenge VNG is requesting us to make a sub-category for so called "citizen cards"; a subset of wallets specifically designed for local services.

From a community perspective I get why it is interesting to separate them from other types of wallets, but from a challenge perspective I rather keep them together. I thought I heard something about a sub-challenge feature. Is that still on? 

I think we will see this sort of discussion more often in the future. If, for example we launch a challenge for smart public transportation, there might eventually be a need to cluster them in categories such as "air versus ground mobility" or "autonomous versus automated vehicles".

**Comment by techsmyth** [comment, marked as answer]
The server actually supports recursive challenges, so there can be challenges within challenges etc - we just simply have not had any bandwidth to expose this on the client.

The other option is to use tags and have some filtering client side on the tags? Much smaller / easier and may actually be a better end experience.

**Comment by techsmyth** [comment]
Re tags, good to also have some consistency. The tags can be seen via:
query {
  ecoverse(ID: "digicampus") {
    nameID
    challenges {
      nameID
      tagset {
        tags
      }
    }
    }
}
